internal/venice-client-common/src/main/java/com/linkedin/venice/stats/AbstractVeniceStats.java

```diff
@@ -32,6 +32,10 @@ public MetricsRepository getMetricsRepository() {
     return metricsRepository;
   }
 
+  protected boolean isTotalStats() {
+    return name.equals(STORE_NAME_FOR_TOTAL_STAT);
+  }
```

Comment:
Nitpick: Since name and `STORE_NAME_FOR_TOTAL_STAT` are final, can we have a `final boolean totalStats` property as well?